cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/config/PdfPageLayout.java

```diff
@@ -16,7 +17,10 @@
     private float marginTop;
     private float marginBottom;
     private float lineDistance;
+    private float header;
     private float footer;
+    private ReportFooterOnPages footerOnPages;
+    private ReportFooterOnPages headerOnPages;
```

Comment:
Well, at a certain point in the code there is no distinction between header and footer, and so ReportFooterOnPages applies to both